.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -33,6 +33,8 @@ If applicable, add code snippets to help explain your problem.
  - NodeJS version (`node -v`):
  - npm or yarn version (`npm -v`):
  - whatsapp-web.js version:
-
```

Comment:
I think you're missing a line break here- if you look at the preview it seems like "Other" is added to the `whatsapp-web.js version` line

Reply:
seems correct